README.md

```diff
@@ -207,6 +207,19 @@ If Doxygen and graphviz are available, the process can also build the documentat
 
 You can also selectively enable/disable other features (e.g., specific plugins you don't care about, or whether or not you want to build the recordings post-processor). Use the --help option when configuring for more info.
 
+### Building on FreeBSD
+* *Note*: rtp_forward of streams only works streaming to IPv6,
+because of [2051](https://github.com/meetecho/janus-gateway/issues/2051) and thus the feature
+is not supported on FreeBSD at the moment
+
+
+When building on FreeBSD you can install the depencencies from ports or packages, here only pkg method is used. You also need to use gmake instead of make,
+since it is a GNU makefile. ./configure can be run without arguments since the default prefix is /usr/local which is your default LOCALBASE.
+Note that the configure.ac is coded to use openssl in base. If you wish to use openssl from ports or any other ssl you must change configure.ac accordingly.
```

Comment:
The `./configure`, `make`, `gmake`, `/usr/local`, `LOCALBASE` and `configure.ac` texts should be in inverted quotes, to follow the style we use in the README.

Reply:
fixed